exercises/Stephen_Smith/ex02_oo_basics/include/Shapes.h

```diff
@@ -0,0 +1,218 @@
+#ifndef SHAPES_H_
+#define SHAPES_H_
+
+#include <string>
+#include <vector>
+#include <memory>
+#include <iostream>
```

Comment:
In the mantid code we would never include `<iostream>` in a header as it increases compile times a lot and having it in a header means it could spread quite easily.

Anything using it should be definined in a `.cpp` file.